src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.awt.*;
+
+public class Ball {
+    private String color;
+    private int number;
+
+
+
+
+    //geter seter col
+    public String getColor() {
+        return color;
+    }
+    public void setColor(String color) {
+        this.color = color;
+    }
+    //get set num
+    public int getNumber() {
+        return number;
+    }
+    public int setNumber(int number) {
```

Comment:
Your `setNumber` method is declared to return `int`, but it doesn't return anything. Setter methods generally do not return a value and have a `void` return type. Please correct this.